src/main/java/uk/gov/di/ipv/cri/common/library/domain/SessionRequest.java

```diff
@@ -22,6 +22,11 @@ public class SessionRequest {
     private SharedClaims sharedClaims;
     private String persistentSessionId;
     private String clientSessionId;
+    private int attemptCount = 0;
+    // TODO: Review the following variables and weather they are needed for passport functionality
```

Comment:
I would prefer if the variables can be reviewed with a view to determining if they are needed prior to merging this PR and then this TODO can be removed